src/jose_curve25519_otp.erl

```diff
@@ -0,0 +1,73 @@
+%% -*- mode: erlang; tab-width: 4; indent-tabs-mode: 1; st-rulers: [70] -*-
+%% vim: ts=4 sw=4 ft=erlang noet
+%%%-------------------------------------------------------------------
+%%% @author Brett Beatty <brettbeatty@gmail.com>
```

Comment:
LGTM, this project was originally licensed under [MPL 2.0](https://tldrlegal.com/license/mozilla-public-license-2.0-(mpl-2)), which was file-based licensing, but it has since been relicensed as [MIT](https://tldrlegal.com/license/mit-license).

In either case: I have no idea whether it's appropriate or not, but I included your name anyway 😄